### PR TITLE
the factory must grow the factory must grow the factory must grow the…

### PR DESCRIPTION
**Pull request by silvereengames**
the factory must grow the factory must grow the factory must grow the factory must grow the factory must grow the factory must grow the factory must grow the factory must grow the factory must grow the factory must grow the factory must grow the factory must grow the factory must grow the factory must grow the factory must grow the factory must grow the factory must grow the factory must grow the factory must grow the factory must grow the factory must grow

### CODE REVIEW

**Comment by HankMH** [comment]
what

File: js/subtitle.js [hunk 1]
```diff
@@ -78,6 +81,8 @@ function getRandomGreeting() {
     "Mission failed, we will get them next time",
     "NUKE INCOMING!",
     "I spent too much time making these",
+    "<h1>E</h1>",
```
Comment (3kh0) [inline js/subtitle.js:84]: please do not xss

File: misc.html [hunk 2]
```diff
@@ -27,7 +27,7 @@ <h1>about:blank</h1>
     </div>
     <div style="width: 100%; max-width: 600px;">
       <div>
-        <h1>Tab Cloaker</h1>
+        <h1><a href="javascript:alert('Easter egg placed by yours truely. \n -Silvereen 🥚')" style="text-decoration: none; color: white;">T</a>ab Cloaker</h1>
```
Comment (3kh0) [inline misc.html:30]: not needed

File: projects.html
Comment (3kh0) [inline projects.html:1]: what is actually being changed here? it seems only that spacing is being modified

**Review by 3kh0 — CHANGES_REQUESTED** [review]
Please see comments attached

File: projects.html
Comment (HankMH) [reply projects.html:1]: dude im just trying to get games for school what does this mean

File: js/subtitle.js (hunk 1 above)
Comment (silvereengames) [reply js/subtitle.js:84]: pls do

File: misc.html (hunk 2 above)
Comment (silvereengames) [reply misc.html:30]: yes it is...

File: projects.html
Comment (silvereengames) [reply projects.html:1]: i dont fucking know merge the pr already

**Comment by 3kh0** [comment]
no

**Pull request closed without merging**